decidim-proposals/config/locales/en.yml

```diff
@@ -116,6 +124,19 @@ en:
           select_a_category: Please select a category
           send: Send
           title: Edit proposal
+        endorsement_button:
+          already_endorsed: Already endorsed
+          already_endorsed_hover: Unendorse
+          endorse: Endorse
+        endorsement_identities_cabin:
+          endorse: Endorse
+        endorsements_count:
+          count:
+            one: ENDORSEMENT
```

Comment:
Should these strings be capitalized via CSS? It's purely a design thing, at removes complexity for translators.

Reply:
"VOTE"/"VOTES" is capitalized like this, I would keep the same for endorsements.

Reply:
Fair enough